libs/libc/obstack/Kconfig

```diff
@@ -0,0 +1,14 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config OBSTACK
```

Comment:
since the linker can optimize the unused function, how about we remove OBSTACK option?

Reply:
Does that also apply to the kernel build? Just asking.. I have never played around with kernel build apart from the applications.

Anyway, it is easier for me to have it in the default, so I prefer that.